common.mk

```
@@ -1390,6 +1390,22 @@ $(srcdir)/ext/etc/constdefs.h: $(srcdir)/ext/etc/depend
 	$(exec) $(MAKE) -f - $(mflags) \
 		Q=$(Q) ECHO=$(ECHO) top_srcdir=../.. srcdir=. VPATH=../.. RUBY="$(BASERUBY)"
 
+KPEG = $(srcdir)/.bundle/bin/kpeg
+$(KPEG): main
+$(KPEG):
+	$(XRUBY) -C "$(srcdir)" bin/gem install --no-document \
+		--install-dir .bundle --conservative --silent "kpeg"
+
+generate-rdoc: $(srcdir)/lib/rdoc/markdown.rb $(srcdir)/lib/rdoc/markdown/literals.rb
+
+$(srcdir)/lib/rdoc/markdown.rb: $(srcdir)/lib/rdoc/markdown.kpeg $(KPEG)
+	$(ECHO) generating $@
+	$(XRUBY) -C "$(srcdir)" $(KPEG) -fs -o lib/rdoc/markdown.rb lib/rdoc/markdown.kpeg
```

`KPEG` is now `$(srcdir)`-prefixed.
Here should be `.bundle/bin/kpeg`, or the prefix needs to be moved from `KPEG` definition to the dependency.
```suggestion
	$(XRUBY) -C "$(srcdir)" .bundle/bin/kpeg -fs -o lib/rdoc/markdown.rb lib/rdoc/markdown.kpeg
```